locale/en.po

```diff
@@ -5336,9 +5333,6 @@ msgstr "Patch Map"
 msgid "THRIVE_LICENSES"
 msgstr "Thrive Licenses"
 
-msgid "THYLAKOID"
```

Comment:
Why is thylakoid now gone? And it doesn't seem like anything replaced this.

Reply:
It's been replaced by chromatophore I think

Reply:
Then that's wrong because the thing in the game is called a thylakoid, meaning that chromatophore is an old name that should be scrubbed from everywhere. You don't have to do that in this PR (and I don't think it fits logically here), but you should absolutely make sure that the new tooltips use the proper name of the translation key.

Reply:
Oh it actually didn't get replaced, it just is in prular (THYLAKOIDS) instead of singular for some reason